src/app/my-account/my-account/my-account.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { OffersBaseOffer } from '@app/core/api';
+import { MyAccountPersonalData } from '../../../api/model/my-account-personal-data';
+import { MyAccountService } from '../my-account.service';
+
+@Component({
+  selector: 'app-my-account',
+  templateUrl: './my-account.component.html',
+  styleUrls: ['./my-account.component.scss'],
+})
+export class MyAccountComponent implements OnInit {
+  public myAnnouncements: Array<OffersBaseOffer> = [];
+  public myAccountPersonalData: MyAccountPersonalData | null | undefined;
+
+  constructor(private myAccountService: MyAccountService) {}
+
+  public ngOnInit() {
+    this.myAccountService.getPersonalData().subscribe((data: MyAccountPersonalData) => {
```

Comment:
`public myAccountPersonalData: Observable<MyAccountPersonalData>`

`this.myAccountPersonalData$ = this.myAccountService.getPersonalData()`

in html
`[myAccountPersonalData]="myAccountPersonalData | async"`

Reply:
done